impeller/aiks/canvas.cc

```diff
@@ -172,23 +172,24 @@ void Canvas::DrawPath(const Path& path, const Paint& paint) {
 }
 
 void Canvas::DrawPaint(const Paint& paint) {
-  if (xformation_stack_.size() == 1 &&  // If we're recording the root pass,
```

Comment:
@bdero note that I got rid of this criteria.  I don't think think it's important that it's the root pass, just that the render target for the render pass is covered.  Let me know if there is something I'm missing.